ethereum/test/beefy-helpers.js

```diff
@@ -40,17 +44,110 @@ async function createBeefyValidatorFixture(numberOfValidators) {
 }
 
 async function createRandomPositions(numberOfPositions, numberOfValidators) {
+
+  const positions = [];
+  for (i = 0; i < numberOfValidators; i++) {
+    positions.push(i);
+  }
+
+  const shuffled = _.shuffle(positions)
+
+  return shuffled.slice(0, numberOfPositions)
+}
+
+async function createFullPositions(numberOfValidators) {
   const positions = [];
-  while (positions.length < numberOfPositions) {
-    const position = Math.floor(Math.random() * numberOfValidators);
-    if (positions.indexOf(position) === -1) {
-      positions.push(position);
+  for (let i = 0; i < numberOfValidators; i++) {
+    positions.push(i);
+  }
+  return positions.sort((a, b) => b - a)
```

Comment:
Why do we sort those positions?

Reply:
this is no longer used, will remove in next PR: https://github.com/Snowfork/snowbridge/pull/454